web/index.html

```diff
@@ -720,6 +724,7 @@ <h4>Discussions</h4>
         found. But due to the probablistic nature of sampling, which cannot guarantee replication
         across successive generations, we need to collect the full set of samples for every problem
         in order to properly estimate the pass@k metrics for various values of k.
+        One can definitely sample until success and calculate the empiricial expected k to compare with average rollout, but we suffice with estimating the pass@k metric.
```

Comment:
We could alternatively sample until success and calculate the empiricial expected k to compare with average rollout, but we suffice with estimating the pass@k metric.